### ISSUE TITLE
Mapping to MSC 2010

### ISSUE THREAD

**Issue by SArndt-TIB** [runnwerth/MSC2020_SKOS#6]
Design decisions (to be discussed!) for mapping from MSC 2020 to older MSC versions:

- only describe relation/ changes between 2020 and 2010
- otherwise keep close to the modelling decisions of MSC 2010, which are (cf. [1]):

> - use subproperties of skos:mappingRelation to link concepts of a newer version of MSC to an older version of MSC
> - concept is unchanged between versions --> skos:exactMatch
> - reclassifications within an area --> skos:relatedMatch (`<example: http://msc2010.org/resources/MSC/2010/05E40> skos:relatedMatch <http://msc2010.org/resources/MSC/2000/05E15>, <http://msc2010.org/resources/MSC/2000/05E20>, <http://msc2010.org/resources/MSC/2000/05E25>`) --> in the database export this is probably (Z)usammenführung (n:1), is probably also used for (A)ufspaltung (1:n)
> - diversification of areas --> skos:broadMatch --> my interpretation: relation between a newly-introduced concept and the older version of its current skos:broader, in the database export this is the change category (N)eu

There are further change markers, not described by [1]:
- skos:closeMatch --> used between a newer concept and an older concpet whose description text changed, older concept has the `dcterms:description "description text changed"`, but apparently no change in position, or splits or merges
- (V)erschiebung --> I do not know which property to use for this one, I guess skos:relatedMatch works, since moving concepts sometimes also incorporates merging concepts (e.g. 12-08 (2020) --> 12Y05 (2010), 12Yxx (2010), 12Y99 (2010), but it seems that moves of concepts have also been expressed by skos:closeMatch, so I am not sure, when to use which
- deletions are not marked in any way, owl offers deprecation

[1] E. Simperl et al. (Eds.): ESWC 2012, LNCS 7295, pp. 763–777, 2012.

**Comment by pdfion** [comment]
There should be clarity of whether a code is a hold-over from MSC2010, a modification of an MSC2010 entry, or a completely new code.  I think time-stamping of the status is relevant.

**Comment by SArndt-TIB** [comment]
I added all historical links between MSC 2010 and MSC 2020 and modelled those with subproperties of skos:mappingRelation as suggtested by the paper quoted above and the MSC 2010 SKOS version. 

Where there were changes in the category name, its description or references, I made a statement on the MSC 2020 concept with dc:description.
If a concept was newly introduced, I stated this as well. 

When a MSC 2020 concept was not taken over or fused with another concept from 2010, I made a statement about the MSC 2010 statement with dc:description.

All statements I made, are based on a detailed comparison of the the prefLabels, the descriptive text and all the references of the concepts.
